authbroker_client/middleware.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, get_response):
     def __call__(self, request):
 
         if resolve(request.path).app_name != 'authbroker_client' and not request.user.is_authenticated:
-            return redirect('authbroker:login')
+            return redirect('authbroker_client:login')
```

Comment:
@lgarvey my attempt to fix this.
https://uktrade.atlassian.net/jira/software/projects/LST/boards/153?selectedIssue=LST-58